### DISCUSSION TITLE
RAG APPLICATION

### DISCUSSION THREAD

**Discussion by kashh56** [kivy/kivy-github-discussions#48] [Q&A]
hey ...i am new here i dont have any prior experience to building applications , I have basic knowledge about FLask, django , html ,css , ai/ml deep learning rag based applications, I want to build a rag based cross platform application that uses custom data for fashion recommendation on the users query.. it uses open source llm and embedding model , its basically RAG based can i build this using flet or any other framework will be better i want to deploy it as well on the playstore and appstore , i only know python ...

**Comment by AndreMiras** [comment]
Hi Kashh56, welcome :wave:
Yes any mobile framework including Kivy should do it. Take whatever you feel the most comfortable with.

**Reply by kashh56** [reply]
@AndreMiras  do u have any experience with such apps..

**Reply by AndreMiras** [reply]
Such a long time ago that it is no longer relevant unfortunately.
Also try your luck on Discord if you're stuck

**Reply by kashh56** [reply]
ok sure Andre thanks for quick replies , if i can help in any way let me know i can contribute to projects..
